iOS Example Frame/iOS Example Frame/MainViewController.swift

```diff
@@ -22,65 +22,100 @@ class MainViewController: UIViewController, CardViewControllerDelegate, ThreedsW
     // Step1 : create instance of CheckoutAPIService
     let checkoutAPIService = Frames.CheckoutAPIService(publicKey: "pk_test_6e40a700-d563-43cd-89d0-f9bb17d35e73",
                                                        environment: .sandbox)
+    var cardViewController: CardViewController?
+
+    // MARK: View Methods.
+
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
         customizeNavigationBarAppearance()
         navigationController?.isNavigationBarHidden = true
     }
 
-    @IBAction func goToPaymentPage(_ sender: Any) {
-        cardViewController.isNewUI = false
-        navigationController?.pushViewController(cardViewController, animated: true)
+    override func viewDidAppear(_ animated: Bool) {
+        let country = Country(iso3166Alpha2: "GB", dialingCode: nil)
+        cardViewController?.addressViewController.setCountrySelected(country: country)
     }
-    
-    @IBAction func goToNewPaymentPage(_ sender: Any) {
-        cardViewController.isNewUI = true
-        navigationController?.pushViewController(cardViewController, animated: true)
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        UIFont.loadAllCheckoutFonts
+        // Do any additional setup after loading the view, typically from a nib.
+
+        let cardFormData = defaultCardFormData()
+        cardViewController = createCardViewController(checkoutAPIService: checkoutAPIService,
+                                                      billingFormData: cardFormData.billingForm,
+                                                      billingFormStyle: cardFormData.billingFormStyle)
+        cardViewController?.delegate = self
```

Comment:
The delegate is also being set on the `create` method.

Reply:
This is in the main host project. (out of current scope) most of old code will be refactored later.